stix_shifter_modules/crowdstrike_logscale/stix_translation/transformers.py

```diff
@@ -0,0 +1,30 @@
+from stix_shifter_utils.stix_translation.src.utils.transformers import ValueTransformer
+from stix_shifter_utils.utils import logger
+import re
+
+LOGGER = logger.set_logger(__name__)
+
+
+class FormatMacAddress(ValueTransformer):
+    @staticmethod
+    def transform(mac_value):
+        """correcting mac address presentation, it should be 6 octate separated
+         by only colon (:) not by any other special character """
+        mac_value = re.sub("[^A-Fa-f0-9]", "", mac_value)
+        mac_length = len(mac_value)
+        if mac_length < 12:
+            for i in range(mac_length, 12):
+                mac_value = "0" + mac_value
```

Comment:
AS per STIX cyber observables, this has been added.

Reply:
Are you referring to "which MUST include leading zeros for each octet."? Can you point to the particular documentation that suggest filling the MAC address with fake information?

I don't agree here. The problem with filling this field with 0 values is that you are creating something that doesn't exist. 

In my opinion, if the field isn't a valid Mac address with 6 octets, you should just fail it and continue as is. It's better to have it empty rather than giving false information, at least imo.